releasenotes/notes/enable-live-data-b53b1000997845da.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    The qiskit-ibmq-provider package now supports IBM Quantum LiveData features.
+    These features allow users to observe the real-time behavior of IBM Quantum
+    backends while executing jobs. Specifically, the provider now includes a
+    new tab in the backend Jupyter-related widget and supports the execution of
+    jobs (via `run()` or `execute()` methods) with the `live_data_enabled=True`
```

Comment:
We should add the doc links for run and execute methods here. It'll look something like :meth:`qiskit.ibmq.providers.IBMQBackend.run`. You can refer to other release notes as example.